openassessment/xblock/static/sass/oa/utilities/_shame.scss

```diff
@@ -71,7 +71,16 @@
     line-height: 0;
   }
 
-
+  // submission answer leaderboard fix (TNL-4009). 4 class selectors is more specific than
+  // lms/static/sass/course/courseware/_courseware.scss#L453 in edx-platform (3 class selectors)
+  .leaderboard__answer .submission__answer__display__content {
+    h2.submission__answer__part__text__title {
+      @include text-align(left);
+      @include float(none);
+      font-size: inherit;
+      margin: 0;
+    }
+  }
```

Comment:
I don't see anything particularly shameful about this css. I would probably put it in _oa-base.scss, but I don't feel strongly about it.